content/04.results.md

```diff
@@ -174,12 +175,38 @@ We performed geneset variant analysis (GSVA) for Hallmark cancer gene sets, demo
 
 <!-- TODO: Add immune deconvolution figure 4C information. -->
 
-#### Telomerase activity 
+![**Figure 4. Transcriptomic overview** A, First two dimensions from UMAP of sample transcriptome data.
+Points are colored by `cancer_group` of the samples they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by `cancer_group`. C, <!-- TODO: Add description about immune deconvolution image we use as main. -->](images/main/Figure4.png){#fig:Fig4}
+
+
+#### Prediction of _TP53_ oncogenicity and telomerase activity
+To understand the _TP53_ phenotype in each tumor, we ran a classifier previously trained on TCGA gene expression data[@doi:10.1016/j.celrep.2018.03.076] to infer _TP53_ inactivation status.
+Using high-confidence SNVs, CNVs, SVs, and fusions in _TP53_ as true positive alterations, we achieved a high accuracy (AUROC = 0.85) compared to randomly shuffled _TP53_ scores (**Figure {@fig:Fig5}A**) in the stranded RNA-Seq dataset. 
+The classifier did not perform well on the poly-A sample subset (**Figure {@fig:FigS5}A-B**), likely due to the low number of poly-A samples in our dataset (N = 58), as we previously reported high accuracy of this classifier on another poly-A dataset [@doi:10.1016/j.celrep.2019.09.071].
```

Comment:
I am not sure how to evaluate this "likely" statement, as I am not familiar enough with the mechanics of the classifier. Is there evidence to suggest that sample size plays a role, or might it just be that these data are sufficiently different from the previous data for some other reason that might cause the classifier to fail?

My other comment, here and elsewhere, is whether we want to refer to previous work with "we" and "our"; while there is certainly overlap in the author lists, it seems like we might be better off keeping those statements more of the type "a previous study reported high accuracy..."